contents/blog/story-about-pivots.md

```diff
@@ -85,11 +85,11 @@ After hundreds of messages to potential users, we eventually got a single custom
 
 Back to basics - what was the actual problem we were solving?
 
-It was the prioritization of where to focus your sales efforts. If 90% of your deals deals won't close, you need to get good at not spending time on those that aren't going to close.
+It was the prioritization of where to focus your sales efforts. If 90% of your sales deals won't close, you need to get good at not spending time on those that aren't going to close.
 
 [Andrew Grove](https://en.wikipedia.org/wiki/Andrew_Grove) has an excellent book, [High Output Management](https://www.amazon.com/High-Output-Management-Andrew-Grove/dp/0679762884/ref=sr_1_1?dchild=1&keywords=high+output+management&qid=1610712757&s=books&sr=1-1). The premise is that your 1:1 meetings with your direct reports are your most leveraged time.
 
-Yet, many managers in practise don't prepare, at all.
+Yet, many managers in practice don't prepare, at all.
```

Comment:
I'm not quite sure about this one, `in practise` does not feel right, but I could be wrong